htmd/queues/lsfqueue.py

```diff
@@ -21,31 +21,40 @@ class LsfQueue(SimQueue, ProtocolInterface):
 
     Parameters
     ----------
+    version : [9, 10], int, default=9
+        LSF major version
     jobname : str, default=None
         Job name (identifier)
-    queue : str or list of str, default=None
-        The queue or list of queues to run on. If list, it attempts to submit the job to the first queue listed. If that
-        queue cannot be used, the next queue listed is considered and this process is repeated.
+    queue : list, default=None
+        The queue or list of queues to run on. If list, it attempts to submit the job to the first queue listed
     app : str, default=None
         The application profile
     ngpu : int, default=1
         Number of GPUs to use for a single job
+    gpu_options : dict, default=None
+        Number of GPUs to use for a single job(dict format: {['mode': ['shared', 'exclusive_process'],]['mps': ['yes',
+        'no'],]['j_exclusive': ['yes', 'no']]})
```

Comment:
Hm, no sorry for not mentioning this earlier I didn't have time :( This is very confusing for me. I understand the idea but the `[ ]` just make me think of lists in python, so I'd try to create a dict of lists. Maybe just point to the LSF docs for available options and say to put them in a dict? Otherwise we could make a "class" for it but I'd prefer not

Reply:
well, the LSF docs will not explain the dictionary format.

The only solution I see for this right now is split the `gpu_options` into three string arguments.

Or wait, what about this forma?:
```
(valid dict entries: {'mode': <'shared' or 'exclusive_process'>}, {'mps': <'yes' or 'no'>},
 {'j_exclusive': <'yes' or 'no'>})
```

Reply:
yeah reads slightly better for me

Reply:
fixed